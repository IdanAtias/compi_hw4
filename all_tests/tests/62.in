C x=3 C;
F y=1 F;
x=y;
