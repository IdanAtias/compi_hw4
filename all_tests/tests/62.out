-- end scope --
var:x:0:CELSIUS
var:y:1:FAHRENHEIT
